SETUP.md

```diff
@@ -134,24 +133,57 @@ SPARK_WORKER_OPTS="-Dspark.worker.cleanup.enabled=true, -Dspark.worker.cleanup.a
 * Runtime version 4.1 (Apache Spark 2.3.0, Scala 2.11)
 * Python 3
 
-### Repository upload
-We need to zip and upload the repository to be used in Databricks, the steps are the following:
-* Clone Microsoft Recommenders repo in your local computer.
-* Zip the contents inside the Recommenders folder (Azure Databricks requires compressed folders to have the .egg suffix, so we don't use the standard .zip):
-```
-cd Recommenders
-zip -r Recommenders.egg .
-```
-* Once your cluster has started, go to the Databricks home workspace, then go to your user and press import.
-* In the next menu there is an option to import a library, it says: `To import a library, such as a jar or egg, click here`. Press click here.
-* Then, at the first drop-down menu, mark the option `Upload Python egg or PyPI`.
-* Then press on `Drop library egg here to upload` and select the the file `Recommenders.egg` you just created.
-* Then press `Create library`. This will upload the zip and make it available in your workspace.
-* Finally, in the next menu, attach the library to your cluster.
-
-To make sure it works, you can now create a new notebook and import the utilities:
+### Repository installation
+You can setup the repository as a library on Databricks either manually or by simply running an installation script. 
+
+
+<details>
+<summary><strong><em>Quick install</em></strong></summary>
+
+> This method only works for **Azure** Databricks.
+
+Prerequisite
+* Install [Azure Databricks CLI (command-line interface)](https://docs.azuredatabricks.net/user-guide/dev-tools/databricks-cli.html#install-the-cli)
+and setup CLI [authentication](https://docs.azuredatabricks.net/user-guide/dev-tools/databricks-cli.html#set-up-authentication).
+
+1. Start a target cluster and copy the target cluster id. Cluster id can be found with following script:
+    ```
+    databricks clusters list
+    
+    <CLUSTER_ID> <CLUSTER_NAME> <STATUS>
```

Comment:
probably worth re-iterating that STATUS must be 'RUNNING' for this script to work, if it isn't you can start it with ```databricks cluster start --cluster-id <CLUSTER_ID>```

Reply:
Oh. thought this comment is for script. Will update the SETUP.md right now!